webpack/assets/javascripts/react_app/components/AutoComplete/components/AutoCompleteFocusShortcut.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import UUID from 'uuid/v1';
+import cx from 'classnames';
```

Comment:
classnames is not in our package.json, should we be included.

Reply:
@sharvit had suggested we should use it :)

Reply:
Added :)